lib/p2p/NodeList.ts

```diff
@@ -40,6 +41,31 @@ class NodeList extends EventEmitter {
     super();
   }
 
+  /**
+   * Set status of peer that bans/unbans and persist in DB
+   */
+  public persistBannedBy = async (nodePubKey: string, bannedBy = true) => {
```

Comment:
I don't think we need both a `persistBannedBy` and a `setBannedBy` method, as we're always calling both of them sequentially. I think it'd be cleaner to just have a `setBannedBy` method that updates both the db and the internal list of nodes that have banned us.